datalad/tests/test_utils.py

```diff
@@ -923,37 +923,47 @@ def test_get_dataset_root(path=None):
         eq_(get_dataset_root(fname), os.curdir)
 
 
-@skip_if_on_windows
-def test_path_startswith():
-    ok_(path_startswith('/a/b', '/a'))
-    ok_(path_startswith('/a/b', '/a/b'))
-    ok_(path_startswith('/a/b', '/a/b/'))
-    ok_(path_startswith('/a/b/', '/a/b'))
-    ok_(path_startswith('/a/b', '/'))
-    ok_(path_startswith('/aaa/b/c', '/aaa'))
-    nok_(path_startswith('/aaa/b/c', '/aa'))
-    nok_(path_startswith('/a/b', '/a/c'))
-    nok_(path_startswith('/a/b/c', '/a/c'))
+def _p(p: str) -> str:
+    """A helper to code paths as POSIX paths in tests  below. Would prepend fake drive
+       C: to absolute paths on Windows"""
+    if on_windows:
+        pm = p.replace('/', os.sep)
+        if p.startswith('/'):
+            return "C:{pm}"
```

Comment:
I think its just a typo?

```suggestion
            return f"C:{pm}"
```

Reply:
tests pass with that fix

Reply:
oh hoh hoh, Thank you @adswa!

Reply:
I think you resolved this comment, but the fix hasn't yet made it into a commit

Reply:
it made it! But then I managed to force push local fix up of a name which I rewrote but without your commit!

Let's commit it again , and thanks again!!!